package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Mosaic chains is an executable to run and manage mosaic chains.",
   "main": "index.js",
   "scripts": {
-    "lint": "eslint src -c .eslintrc.json --ext ts",
+    "lint": "eslint src -c .eslintrc.json --fix --ext ts",
```

Comment:
Should we add `--fix`. or we should do it manually?

Reply:
Shouldn't be there. By mistake 😞